.github/workflows/axes_selector.yml

```diff
@@ -260,8 +256,8 @@ jobs:
             core.setOutput('JACOCO', '${{ inputs.JACOCO }}');
             core.setOutput('DB_TESTS', '${{ inputs.DB_TESTS }}');
             core.setOutput('PERFORMANCE', '${{ inputs.PERFORMANCE }}');
-            if (${{ inputs.DB_TESTS }}) {
-              const enabled_databases = JSON.parse(${{ inputs.enabled_databases }});
+            if ('${{ inputs.DB_TESTS }}' === 'true') {
```

Comment:
Please can I ask if this is intentional? And if so is it related to the JDK versions change? No problem for me it being in here and in this commit even but just good to know if it's related. Maybe if it's not related to JDK versions the commit message could be expanded?

Reply:
Please let me know what you think/suggest and I'll apply it to this PR. Thanks for raising this discussion

Reply:
I personally think it should not be required and feel comfortable to treat it as a boolean, having seen it is declared as a boolean: https://github.com/jbosstm/narayana/blob/e660b5a0242a5ca74d9f24cdb3d9d58ff8b6a21d/.github/workflows/axes_selector.yml#L59 - but you are more familiar with this than me so if you think it's safer then I think we should go with what you think